src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -72,19 +72,22 @@ export default App.extend({
 
     if (storedState) {
       this.setState(storedState);
+      this.getState().setClinicianId(this.clinicianId);
       this.startFiltersApp();
       return;
     }
 
     this.setState({ id: this.worklistId });
 
+    this.getState().setClinicianId(this.clinicianId);
```

Comment:
Maybe this is ok, but having two sets in a row means we are hitting the localstorage twice.. This would be better in a single set if possible

Reply:
What's the downside to hitting local storage twice?

I'm not quite sure how to implement it in a way that sets it once (yet). It seems like we have to do it in a second call because we have to check for `if (this.clinicianId)` before changing the state. But I can come back to this after I get the link/s added in the UI.

Reply:
what's wrong with:
```js
initState(state) {
  if (this.clinicianId) { 
    this.setState(extend({}, state, { clinicianId: this.clinicianId });
    return;
  }
  this.setState(state);
} 
```
or if/else 🤮 
```js
  if (this.clinicianId) {  this.setState({ id: this.worklistId, clinicianId: this.clinicianId }); } else { this.setState({ id: this.worklistId }); }
```

it's likely ok to hit localstorage a bunch.. and because it's before the start we _can_ probably hit the state twice..  it's not _ideal_ to do two subsequent `change` events but it's almost more ideological at this point.. there's no particular issue right now if we want to hit it twice...  but there'd certainly be other similar cases where we _couldn't_ do two subsequent state changes without it rendering twice or something like that..